README.md

```diff
@@ -65,62 +59,49 @@ Then following the prompts in your terminal to create the project structure.
 
 ## Installing pre-commit hooks
 
-This repo uses the Python package [`pre-commit`](https://pre-commit.com) to manage pre-commit hooks. Pre-commit hooks
-are actions which are run automatically, typically on each commit, to perform some common set of tasks. For example, a
-pre-commit hook might be used to run any code linting automatically, providing any warnings before code is committed,
-ensuring that all of our code adheres to a certain quality standard.
+This repo uses the Python package [`pre-commit`](https://pre-commit.com) to manage pre-commit hooks. Pre-commit hooks are actions which are run automatically, typically on each commit, to perform some common set of tasks. For example, a pre-commit hook might be used to run any code linting automatically, providing any warnings before code is committed, ensuring that all of our code adheres to a certain quality standard.
 
 For this repo, we are using `pre-commit` for a number of purposes:
 - Checking for any secrets being committed accidentally;
 - Checking for any large files (over 5MB) being committed; and
 - Cleaning Jupyter notebooks, which means removing all outputs and execution counts.
 
-We have configured `pre-commit` to run automatically on _every commit_. By running on each commit, we ensure
-that `pre-commit` will be able to detect all contraventions and keep our repo in a healthy state.
+We have configured `pre-commit` to run automatically on _every commit_. By running on each commit, we ensure that `pre-commit` will be able to detect all contraventions and keep our repo in a healthy state.
 
 In order for `pre-commit` to run, action is needed to configure it on your system.
 - [Install](#installing-python-packages) the `pre-commit` package into your Python environment from
 `requirements-dev.txt`; and
-- Run `pre-commit install` to set-up `pre-commit` to run when code is _committed_.
+- Run `pre-commit install` in your terminal/shell/console to set-up `pre-commit` to run when code is _committed_.
 
 ### Setting up a baseline for the `detect-secrets` hook (if one doesn't already exist)
 
-The `detect-secrets` hook requires that you generate a baseline file if one is not already present within the root
-directory. This is done via running the following at the root of the repo:
+The `detect-secrets` hook requires that you generate a baseline file if one is not already present within the root directory. This is done via running the following at the root of the repo in your terminal/shell/console:
 
 ```shell script
 detect-secrets scan > .secrets.baseline
 ```
 
-Next, audit the baseline that has been generated by running:
+Next, audit the baseline that has been generated by running the following in your terminal/shell/console:
 
 ```shell script
 detect-secrets audit .secrets.baseline
```

Comment:
Not related to this PR really, but just to note that this brings up a lot of false positives in `renv.lock`, and it doesn't seem to be possible to ignore a file.

Reply:
This is interesting. I though I was able to ignore it by including `renv.lock` in the `.pre-commit-config.yaml` file in this section:
https://github.com/ukgovdatascience/govcookiecutter/blob/feature/r-setup/.pre-commit-config.yaml#L17

Reply:
Aha, that does the trick, thanks.